Example/Backpack/NavigationData.swift

```diff
@@ -126,6 +126,7 @@ class NavigationData: NSObject {
             Item(name: "Labels", value: .story(loadStoryboard(name: "Labels", identifier: "LabelsViewController")))
             Item(name: "Navigation bars", value: .story(loadStoryboard(name: "NavigationBar", identifier: "NavigationBarViewController")))
             Item(name: "Overlay views", value: .story(loadStoryboard(name: "OverlayView", identifier: "OverlayViewViewController")))
+            Item(name: "Map", value: .story(loadStoryboard(name: "Map", identifier: "MapsViewController")))
```

Comment:
Do we really wanna use storyboard for this? With our new way of constructing stories using code is quite simple and doesn't suffer from the potential merge issues with storyboards 🤔

Reply:
I kinda feel like the storyboard is ok for stuff like this, but we just want to avoid using it for nav, right?